src/controllers/confirmation.controller.ts

```diff
@@ -25,20 +26,28 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
   if (!email) {
     return next(createMissingError("User Email"));
   }
-  const token = req.chSession.accessToken();
-  const request = sessionService.getRequest(req.chSession);
-  if (token && request) {
-    try {
-      await apiClient.callProcessorApi(companyNum, token, request.extension_request_id);
-    } catch (e) {
-      logger.error("Error processing application " + JSON.stringify(e));
-      return next(e);
-    }
-    if (activeFeature(process.env.FEATURE_REQUEST_COUNT)) {
-      RequestCountMonitor.updateTodaysRequestNumber(1);
-    } else {
-      logger.info("Feature flag is toggled off for request number counter update");
-    }
+  const isSubmitted: boolean = req.chSession.data[keys.SUBMITTED];
+  if (!isSubmitted) {
+    req.chSession.data[keys.SUBMITTED] = true;
```

Comment:
`req.chSession.data[keys.SUBMITTED]` equates to `submitted`. This provides no context in the global session. We should be putting this within the extensions session or at the very least renaming it to something less ambiguous